src/improvements/tasks/sep/017-1-U16-opcm-upgrade-v400-base-approveHash/VALIDATION.md

```diff
@@ -0,0 +1,109 @@
+cast calldata \
+  "approveHash(bytes32)" \
+  0x4769c1cfaa9cb7b373313abe3fa6469a2a1e756b059a74a141511355ff4f14b2
+
+# This will print out the calldata for the approveHash call:
+# 0xd4d9bdcd4769c1cfaa9cb7b373313abe3fa6469a2a1e756b059a74a141511355ff4f14b2
+
+# Now encode the multicall payload, where `0x0fe884546476dDd290eC46318785046ef68a0BA9` is
+# the address of the L1 ProxyAdmin Owner for Base.
+cast calldata \
+  "aggregate3Value((address,bool,uint256,bytes)[])" \
+  '[(0x0fe884546476dDd290eC46318785046ef68a0BA9,false,0,0xd4d9bdcd4769c1cfaa9cb7b373313abe3fa6469a2a1e756b059a74a141511355ff4f14b2)]'
+```
+
+The resulting calldata:
+
+```text
+0x174dea710000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000fe884546476ddd290ec46318785046ef68a0ba90000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000024d4d9bdcd4769c1cfaa9cb7b373313abe3fa6469a2a1e756b059a74a141511355ff4f14b200000000000000000000000000000000000000000000000000000000
+```
+
+# State Validations
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state
+  changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain
+  Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+### State Overrides
+
+Note: The changes listed below do not include threshold, nonce and owner mapping overrides. These changes are listed and explained in the [NESTED-VALIDATION.md](../../../../../NESTED-VALIDATION.md) file.
+
+### Task State Changes
+
+#### `0x0fe884546476ddd290ec46318785046ef68a0ba9` (Base Operations Safe) - Chain ID: 11763072
+
+- **Key:** `0x3cb894e4fceae49273c232ca8e3f156b944a12b10d7de24e1d7b2e25057d1e8b`
+- **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000`
+- **After:** `0x0000000000000000000000000000000000000000000000000000000000000001`
+- **Summary:** This is setting the approved hash for the Base Operations safe. You can compute the slot using ....
+- **Detail:**
+
+  ---
+
+#### `0x0fe884546476ddd290ec46318785046ef68a0ba9` (Base Security Council Safe) - Chain ID: 11763072
+
+- **Key:** `todo`
```

Comment:
This is not filled out because even when I simulated with `SIMULATE_WITHOUT_LEDGER=1 just --dotenv-path $(pwd)/.env --justfile ../../../nested.just simulate base-council`, the state change logged was incorrectly in the Base Operations Safe (`0x646132a1667ca7ad00d36616afba1a28116c770a`) and not the Base Security Council Safe (`0x5dfEB066334B67355A15dc9b67317fD2a2e1f77f`). Need to debug this, but if you know why this is happening let me know